### PR TITLE
Issues resolved #5

### LINKED ISSUE #5: Create a skeleton component for the code editor
- Create the component where the person will write code.

### PR DESCRIPTION
**Pull request by pratham9770**

### CODE REVIEW

**Comment by pratham9770** [comment]
Code editor with language-specific snippets.
image

File: frontend/src/utils/languages.jsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    java: "15.0.2",
+    csharp: "6.12.0",
+    php: "8.2.3",
+    cpp: "11.1",
+    c: "11.1",
+    ruby: "3.1",
+    go: "1.18",
+    swift: "5.5",
+    kotlin: "1.6",
+  };
```
Comment (ravinder-chadha) [inline frontend/src/utils/languages.jsx:15]: Should we hard code the versions? or is there a better way to do this

File: frontend/src/App.js [hunk 2]
```diff
@@ -1,12 +1,12 @@
...
 function App() {
   return (
-    <div className="App">
-     <Output />
-    </div>
-  );
-}
+    <Box minH="100vh" bg="#0f0a19" color="gray.500" px={6} py={8}>
+      <CodeEditor />
+    </Box>
```
Comment (ravinder-chadha) [inline frontend/src/App.js:9]: Please don't remove your teammates changes, rather add your changes.

**Comment by ravinder-chadha** [comment]
Can you also please try to link the issue #5 again. You need to add it in the description, as it cannot be linked in the title.

File: frontend/src/utils/languages.jsx (hunk 1 above)
Comment (pratham9770) [reply frontend/src/utils/languages.jsx:15]: if the api is only using the latest version we can delete this from here?

File: frontend/src/App.js (hunk 2 above)
Comment (pratham9770) [reply frontend/src/App.js:9]: oh sorry sir i didn't realise

**Pull request closed without merging**